File: Math/codeforces_136A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/136/A
// submission link ===>> https://codeforces.com/contest/136/submission/228041107

#include <iostream>
#include <vector>
using namespace std;

int main()
{
    int n;
    cin >> n;

    vector<int> presents(n + 1);

    for (int i = 1; i <= n; i++)
    {
        int present;
        cin >> present;
        presents[present] = i;
    }

    for (int i = 1; i <= n; i++)
    {
        cout << presents[i] << " ";
    }
    cout << endl;

    return 0;
}

File: Math/codeforces_136A.py

# Problem link ===>> https://codeforces.com/problemset/problem/136/A

def main():
  """Solves the Codeforces A. Presents problem."""

  n = int(input())

  presents = [None] * (n + 1)

  for i in range(1, n + 1):
    present = int(input())
    presents[present] = i

  for present in presents[1:]:
    print(present, end=" ")
  print()


if __name__ == "__main__":
  main()
